flujo/application/core/async_iter.py

```
@@ -1,15 +1,14 @@
+"""Compatibility shim for `async_iter` (moved to core/support/)."""
+
 from __future__ import annotations
 
-import inspect
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from flujo.application.core.support.async_iter import *  # noqa: F401,F403
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)